playbooks/roles/nginx/tasks/add-letsencrypt.yml

```diff
@@ -0,0 +1,76 @@
+---
+
+- name: Installing certbot dependencies
+  apt: name="{{ item }}" update_cache=yes
+  with_items: "{{ le_packages }}"
+  become_method: sudo
+  become: yes
+  become_user: root
+  when: USE_LETSENCRYPT == "yes"
```

Comment:
Jenkins will set value of this var in yes\no. It looks better, then true\false https://monosnap.com/file/93bqg5Rb3euBHFcK5IvdiNOy3BvlW2#